tests/admin_views/tests.py

```diff
@@ -4750,6 +4753,7 @@ def test_readonly_backwards_ref(self):
         response = self.client.get(reverse('admin:admin_views_topping_add'))
         self.assertEqual(response.status_code, 200)
 
+    @ignore_warnings(category=RemovedInDjango20Warning)
```

Comment:
If you could add `# for allow_tags deprecation` besides this line and the others `@ignore_warnings` in this file, that's helpful when removing deprecated features. Thanks!